app/navigation/MDRNavigator.scala

```diff
@@ -66,7 +80,7 @@ class MDRNavigator @Inject() () extends Navigator {
   private def doYouHaveUniqueTaxPayerReference(mode: Mode)(regime: Regime)(ua: UserAnswers): Option[Call] =
     ua.get(DoYouHaveUniqueTaxPayerReferencePage) map {
       case true  => routes.BusinessTypeController.onPageLoad(mode, regime)
-      case false => routes.WhatAreYouRegisteringAsController.onPageLoad(mode, regime)
+      case false => jumpToCYA(mode, regime, routes.WhatAreYouRegisteringAsController.onPageLoad(mode, regime))
```

Comment:
@iservport Can we name this method better

Reply:
Removed